railties/lib/rails/api/task.rb

```diff
@@ -69,7 +73,12 @@ class Task < RDoc::Task
             README.rdoc
             lib/**/*.rb
           ),
-          exclude: "lib/rails/generators/rails/**/templates/**/*.rb"
+          exclude: %w(
+            lib/rails/generators/rails/**/templates/**/*.rb
+            lib/rails/test_unit/*
+            lib/rails/test_help.rb
```

Comment:
Yeah, you're totally right. However, this file added nothing to the API apart from inserting `TestCase < ActiveSupport::TestCase` at the top level in the sidebar (as you can see on the screenshot in this PR's description).

Anyway, this may be useful for people to consult this file on the API site so it's present under the "files" label but there's no more duplicate as there is now a `:stopdoc:` comment in this file.